spec/system/post_index_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
```

Comment:
Great job @DaveZag 🤝

- [ ] As stated in the project requirements, could you implement all the tests as displayed in the screenshot below?
![post-index](https://user-images.githubusercontent.com/63917725/217300030-27268fce-eda1-418c-b009-6af96ec5038e.PNG)

Reply:
@1GORDON Thanks for the feedback. The required changes have been implemented.🙂